### ISSUE TITLE
FPN Snippets with United States Targeting seen in Germany and Switzerland

### ISSUE THREAD

**Issue by jkt230** [mozmeao/snippets-service#1261]
Hi Giorgos, 
Been flagged by Jennifer Balaco (Germany) and Josh Gruen (Switzerland) that they saw FPN snippets when it's supposed to be US only. I think it may have to do with the fact that we chose English (American) as our locale instead of the typical EN-All? 

Here are the job ids:
22026
22027
22028
22029
22030
22031
22032
22033
22034
22035
22036
22037

Also seen here: https://snippets-admin.mozilla.org/list/jobs/?name=fpn&locale=&only_scheduled=true&calendar=false

**Comment by glogiotatidis** [comment]
I confirm that snippets are targeted to US only. Locale does not affect this. I will reach out to Jenn and John to help debug this.

**Comment by glogiotatidis** [comment]
tl;dr This is not a bug in the snippets-service, everything works as intended. It's people that travel that may get country targeted snippets. 

OK so I talked to J and it seems her browser believe it's in US. That explains why she's getting FPN snippets. I'll talk with AS team to check how often the `region` value gets updated. Will leave this open to track the `region` caching issue.

**Comment by glogiotatidis** [comment]
Firefox saves region when the Profile is first created. It's very possible the Jenn created her profile in US, while getting a new laptop.

This is a rather edge case and there's no easy fix according to the AS team. One can reset their profile or manually set the browser.search.region pref to another value to change country if they want.

tl;dr: Snippets are targeted as they should and everything works as expected.

**Comment by jkt230** [comment]
Thanks for investigating Giorgos!
